tests/ragger/snapshots/nanosp

```diff
@@ -0,0 +1 @@
+nanox/
```

Comment:
We can have issues with this, if for example we bump the `NANOX` api_level and the font changes

Reply:
We might get issues, but the day this breaks I'll remove this symbolic link and do a golden run for LNS+. :+1:

Reply:
I'm not sure I see the added value of not doing it now

Reply:
Duplicating all the snapshots for something that might happen is overkill, especially since the day this happens, even with duplicated snapshots for LNX & LNS+, the CI would still break and the snapshot would have to be re-generated anyway for one or the other.
There is no future-proof solution to that problem and at least this one with the symbolic link is lighter.